packages/lesswrong/components/posts/usePostReadProgress.ts

```diff
@@ -63,7 +63,7 @@ export const usePostReadProgress = ({ updateProgressBar, disabled = false, delay
     
     const displayedScrollWindowPercent = clampPct(currentScrollPct / windowBodyPercent);
     const displayedWindowScrollHeight = absoluteScrollWindowHeight * displayedScrollWindowPercent;
-
+    
```

Comment:
remove no-op dif

Reply:
I'm confused because I thought I fixed this (in response to this comment) and shipped the fix. It's fixed on my local branch, but doesn't seem to be here.